dubbo-registry/dubbo-registry-api/src/main/java/org/apache/dubbo/registry/client/InstanceAddressURL.java

```diff
@@ -149,30 +149,30 @@ public String getParameter(String key) {
             return getSide();
         }
 
-        if(consumerParamFirst(key)) {
+        if (consumerParamFirst(key)) {
             URL consumerUrl = RpcContext.getServiceContext().getConsumerUrl();
             if (consumerUrl != null) {
                 String v = consumerUrl.getParameter(key);
-                if (v != null) {
+                if (StringUtils.isNotEmpty(v)) {
```

Comment:
It seems the new logic is different from the origina. `"a":""` will get different result. I'm not sure this pr have changed the author's original intention. 
@AlbumenJ plz help to review it